examples/Billing/AddBillingSetup.cs

```diff
@@ -181,18 +193,25 @@ public override string Description
             // GetBillingSetup.cs for a more detailed example of requesting billing setup
             // information.
             string query = @"
-                    SELECT billing_setup.id, billing_setup.status
-                    FROM billing_setup
-                    WHERE billing_setup.status = 'APPROVED' LIMIT 1";
+                SELECT billing_setup.end_date_time
+                FROM billing_setup
+                WHERE billing_setup.status = 'APPROVED'
+                ORDER BY billing_setup.end_date_time DESC";
 
             PagedEnumerable<SearchGoogleAdsResponse, GoogleAdsRow> searchResponse =
                 googleAdsService.Search(customerId.ToString(), query);
 
             if (searchResponse.Any())
             {
-                // If there is an existing approved billing setup, then you can specify any future
-                // start date for the billing setup.
-                billingSetup.StartDateTime = DateTime.Today.AddDays(1).ToString("yyyyMMdd");
+                // If there are any existing approved billing setups, the new billing setup can
+                // start immediately after the last.
+                string lastEndingDateTimeString = searchResponse.First().BillingSetup.EndDateTime;
+
+                // Check if the existing billing setup has no end date (i.e., is set to run
+                // indefinitely).
+                DateTime lastEndingDateTime = lastEndingDateTimeString == null ? DateTime.Today :
```

Comment:
@AnashOommen Setting a billing setup when there is already one set with indefinite end date may lead to board effect that would prematurely end it. I think we should throw an error when this happens, WDYT?

Reply:
We should.